SCons/Node/Python.py

```diff
@@ -152,12 +152,10 @@ def get_contents(self):
         Get contents for signature calculations.
         :return: bytes
         """
-        text_contents = self.get_text_contents()
-        try:
-            return text_contents.encode()
-        except UnicodeDecodeError:
-            # Already encoded as python2 str are bytes
-            return text_contents
+        contents = str(self.value).encode('utf-8')
+        for kid in self.children(None):
```

Comment:
`child` seems to be in use everywhere else

Reply:
I would prefer `child` as well but it looks like `kid` is still used frequently in SCons/Node/*.